rcsb/workflow/wuw/PdbCsmImageWorkflow.py

```diff
@@ -60,20 +60,19 @@ def imagesGenJpgs(self, **kwargs: dict) -> None:
 
         # generate list of commands
         argsL = []
-        for line in idList:
+        logger.info("Id list contains %s entries", len(idList))
+        for i, line in enumerate(idList):
             name = line.lower()
-            if useIdSubdir:
-                name = os.path.join(name[1:3], name)
 
-            bcifFileName = name + ".bcif.gz"
-            logger.info("Running %s %s %s", name, bcifFileName, contentTypeDir)
+            bcifFileName = os.path.join(idHash(name), name) + ".bcif.gz"
+            logger.info("%s running %s %s %s", i, name, bcifFileName, contentTypeDir)
 
             if contentTypeDir == "pdb":
                 bcifFilePath = os.path.join(pdbBaseDir, bcifFileName)
             else:
                 bcifFilePath = os.path.join(csmBaseDir, bcifFileName)
 
-            outPath = os.path.join(jpgsOutDir, contentTypeDir)
+            outPath = os.path.join(jpgsOutDir, contentTypeDir, idHash(name), name)
```

Comment:
Rather than calling `idHash(name)` twice, I'd suggest calling it once above and assign it to a variable (e.g., `nameHash`), then just referencing that in the `bcifFileName = ...` and `outputPath = ...` lines.